src/NodeApi/JSObject.cs

```diff
@@ -24,6 +29,16 @@ public JSObject() : this(JSValue.CreateObject())
     {
     }
 
+    #region IJSValue<JSObject> implementation
+
+    public static bool CanBeConvertedFrom(JSValue value) => value.IsObject();
```

Comment:
Can static interface methods be explicitly implemented? I ask because these methods are an implementation detail that might be better to hide; the naming is a bit awkward and we want to encourage use of the `Is<>()` / `As<>()` methods instead.

Reply:
Yep, they can be explicitly implemented and I made the `CanCreateFrom` static method private.